content/guides/getting-started/opening-the-app.md

```diff
@@ -15,37 +15,41 @@ title: Opening the App
 
 ## `cypress open`
 
-If you used `npm` to install, Cypress has now been installed to your
+<Alert type="info">
```

Comment:
not even sure we need this alert section and its content

Reply:
Yeah it seems a little excessive. I only kept it and wrapped the alert to tie it to the “Long version” using the full path we discuss below. 

We may not really need that either. I believe it’s only for users not using npm or a script, etc. not clear on how many people would do that. 

TMI? We could just pull these bits.